sys-block/arcconf/metadata.xml

```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person">
-		<email>dev-zero@gentoo.org</email>
```

Comment:
Is this approved by @dev-zero?

Reply:
Yes, see the git commit, it's written there.